product_selection/select_product.py

```diff
@@ -96,10 +96,15 @@ def parse_user_jsons(self, user_who, user_what):
         if 'Price' in self.user_info['what']:
             price = self.user_info['what']['Price']
             price = str(price).replace('\'','').replace('\"', '')
-            
+            print("price:")
+            print(price)
             price_list = [i for i in price[1:-1].split(",") if i.strip()]
             for count, element in enumerate(price_list):
                 price_list[count] = float(re.sub("[^\d\.]", "", element))
+            if len(price_list) == 1:
```

Comment:
I think this may not solve the issue of prices being returned as strings rather than numbers. If it's a string, I think we want to pass it back to chatgpt to return it as an appropriately formatted tuple (numbers rather than strings). I think this will still cause an internal server error if a string is returned.

Reply:
initially i think it is a string then we have to convert it manually to a tuple

Reply:
![image](https://github.com/user-attachments/assets/d1fb0363-5d6d-49c1-a993-ca7390835a4e)